src/keysManager.js

```diff
@@ -34,9 +36,12 @@ export default class KeysManager {
   async generateKeys() {
     return await addressGenerator();
   }
+
   createKeys({mining, voting, payout, sender}){
-    const gasPrice = this.web3_10.utils.toWei('2', 'gwei')
-    return this.keysInstance.methods.createKeys(mining, voting, payout).send({from: sender, gasPrice})
+    let gasPrice = this.web3_10.utils.toWei('2', 'gwei');
+    if (this.netId === constants.NETID_DAI_TEST || this.netId === constants.NETID_DAI) {
+      gasPrice = 0;
```

Comment:
Maybe better to use the same method here `this.web3_10.utils.toWei('0', 'gwei');` to be sure that the representation is correct (BigNumber/String/Number)?

Reply:
It works fine with `gasPrice = 0`, but ok, I've added web3.utils.toWei for zero price too: https://github.com/poanetwork/poa-dapps-keys-generation/pull/82/commits/f39cf526b6dfa874337fb2865fcb00fda4e8d9e0.